File: patterns/javascript-patterns/pattern120.js
// Pattern 120: ...

//   *
//   *
//  **
//  **
// ***
// ***

for (let i = 1; i <= 3; i++) {
    for (let x = 2; x > 0; x--) {
        let spaces = '';
        for (let y = 2; y >= i; y--) {
            spaces += ' ';
        }
        let stars = '';
        for (let j = 0; j < i; j++) {
            stars += '*';
        }
        console.log(spaces + stars);
    }
}

File: patterns/python-patterns/pattern120.py
# Pattern 120: ...

#   *
#   *
#  **
#  **
# ***
# ***

for i in range(1, 4):
    for x in range(2, 0, -1):
        for y in range(2, i - 1, -1):
            print(" ", end="")
        for j in range(i):
            print("*", end="")
        print()
